xml/ns-System.Runtime.Intrinsics.Arm.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Runtime.Intrinsics.Arm">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains classes representing the different SIMD instruction sets available to the ARM architecture. Not all of these instructions sets are available on all ARM CPUs though. Check the "IsSupported" property on each class to determine compatibility with the executing CPU.</summary>
```

Comment:
This, and the same for x86, could likely be reworded slightly. The underlying APIs largely touch SIMD code, but there are also other APIs which aren't SIMD at all.

Perhaps this could say something like:
> Exposes select instructions available on ARM (branding needed?) hardware. These instructions are divided into separate classes representing the underlying ISAs (Instruction Set Architectures) exposed by the platform. Support for an ISA can be queried via the `IsSupported` property exposed on the respective type and may be false if the underlying hardware does not support it or if it is not currently supported by the underlying runtime.

I

Reply:
Some of this feels more like conceptual docs however and it is a bit more wordy than most summaries.

Reply:
I agree. I reworded it, mixing your phrasing with mine. The revised phrasing is able to drop the comment about `IsSupported` if you deem it too wordy still.